### PR TITLE
[THEME] convert theme "wide" in option of theme

### PR DESCRIPTION
**Pull request by abarrau**
Propose this PR for use only list of principal theme 
The "wide" is converted as an options of theme on user account Settings.

list of theme with options

With this way, we can add other "option" for theme and add an overrides_xxx.css file on folder theme.

Remark : i am not change the content of css class for wide

Remark 2 : during testing this PR, if you came back on other branch, dont forget to change the content of "user_stylesheet" field on table users

Stay to do : change the qso/edit_ajax header, after the #2861 will be validated.

### CODE REVIEW

File: application/migrations/165_convert_wide_theme_as_option.php
Comment (HB9HIL) [inline application/migrations/165_convert_wide_theme_as_option.php:1]: down function does not contain the user_stylesheet

**Comment by magicbug** [comment]
While I appreciate the time that has gone into this, I would prefer to keep themes separate for maintainability.

Therefore this won't make it into Cloudlog.

**Comment by AndreasK79** [comment]
@magicbug as the themes are 99% the same, I don't understand why this is any worse to maintain. This PR just adds an option for the theme. 

The only thing that should be fixed, is that is should check if a wide override exists, else you get a 404 if you choose wide for the theme, and it does not have it.

**Comment by HB9HIL** [comment]
@magicbug Maybe it isn't that bad. We could reduce a significant amount of code and the 404 issue is a simple fix (only show available options for theme)

**Pull request closed without merging**